webcompat/templates/shared/footer.html

```diff
@@ -40,33 +39,33 @@
       </ul>
 
 <!-- sub nav for contribute > working with bugs -->
-      {% if request.url_rule.endpoint in ['contributors_report_bug', 'contributors_reproduce_bug', 'contributors_diagnose_bug', 'contributors_site_outreach'] %}
+      {% if 'contributors_bug_' in request.url_rule.endpoint %}
         <ul class="grid-cell x1 footer-sub-list">
-          <li class="footer-sub-item {% if request.url_rule.endpoint == 'contributors_report_bug' %}active{% endif %}">
+          <li class="footer-sub-item {% if request.url_rule.endpoint == 'contributors_bug_report' %}active{% endif %}">
             <a class="footer-item-link" href="/contributors/report-bug">Report a Bug</a>
           </li>
-          <li class="footer-sub-item {% if request.url_rule.endpoint == 'contributors_reproduce_bug' %}active{% endif %}">
+          <li class="footer-sub-item {% if request.url_rule.endpoint == 'contributors_bug_reproduce' %}active{% endif %}">
             <a class="footer-item-link" href="/contributors/reproduce-bug">Reproduce a Bug</a>
           </li>
-          <li class="footer-sub-item {% if request.url_rule.endpoint == 'contributors_diagnose_bug' %}active{% endif %}">
+          <li class="footer-sub-item {% if request.url_rule.endpoint == 'contributors_bug_diagnosis' %}active{% endif %}">
             <a class="footer-item-link" href="/contributors/diagnose-bug">Diagnose a Bug</a>
           </li>
-          <li class="footer-sub-item {% if request.url_rule.endpoint == 'contributors_site_outreach' %}active{% endif %}">
+          <li class="footer-sub-item {% if request.url_rule.endpoint == 'contributors_bug_outreach' %}active{% endif %}">
             <a class="footer-item-link" href="/contributors/site-outreach">Site Outreach</a>
           </li>
         </ul>
       {% endif %}
 
 <!-- sub nav for contribute > other contributions -->
-      {% if request.url_rule.endpoint in ['contributors_build_tools', 'contributors_web_platform_research', 'contributors_organize_webcompat_events'] %}
+      {% if 'contributors_other_' in request.url_rule.endpoint %}
```

Comment:
For this I have to look for all 3 "other" routes, because there's nothing common there to match on:

| (proposed) method_name  | route |
| ------------- | ------------- |
| contributors_other_tools  | /contributors/build-tools |
| contributors_other_research | /contributors/web-platform-research  |
| contributors_other_events |/contributors/organize-webcompat-events|